src/clj/puppetlabs/puppetserver/cli/subcommand.clj

```diff
@@ -52,3 +52,11 @@
       (throw+))
     (finally
       (shutdown-agents))))
+
+(defn jruby-run
```

Comment:
I'm okay with having this be done here as you've done it.  The `System/exit` for non-JRuby based subcommands already lives in this namespace, so it seems reasonable to do the same here.

We've talked in the past about maybe pushing more of the JRuby CLI functions into `jruby-utils`.  If we had other consumers of that besides just Puppet Server, that might still make sense to do.  Doing that now, though, would just lead us to need to do a release of jruby-utils everytime we want to expose a change to Puppet Server, with no code sharing benefits.  That wouldn't seem worthwhile.